app/lib/io/zipUtil.js

```diff
@@ -12,7 +12,26 @@ define([
     var NotImplementedError = errors.NotImplemented;
 
     var unpack = function(async, zipData, io, targetPath){
-        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+        if (async)
+            throw new NotImplementedError('Asynchronous ZIP unpack method is not yet implemented');
+
+        var zip = new JSZip(zipData);
+        var files = zip.files;
+        console.log("unpacking files: ", files);
+        var i;
+
+        for (i in files){
+            var file = files[i];
+            var absolute_path = [targetPath, file.name].join(targetPath[targetPath.length-1]=='/' ? "" : "/");
+
+            if (file.dir){
+                io.mkDir(false, absolute_path);
+            } else {
+                var dir_abs_path = absolute_path.substring(0, absolute_path.lastIndexOf("/"));;
+                io.ensureDir(false, dir_abs_path);
+                io.writeFile(false, absolute_path, file.asBinary());
```

Comment:
What data type is returned by `file.asBinary()`?
Does it work with NodeJS? (Maybe)
Does it work with other IO implementations? I don't think so, We should leave a comment somewhere that not all `io` arguments will work yet!